### DISCUSSION TITLE
User `-configdir` instead of `-config` to start Xorg

### DISCUSSION THREAD

**Discussion by matt335672** [neutrinolabs/xrdp#2556] [Ideas]
@Seas0 has just posted [this comment](/neutrinolabs/xrdp/issues/2010#issuecomment-1430062495) regarding using `-configdir` rather than `-config` to start Xorg. It nicely avoids issues where the default config dir has stuff in which can interfere with `xorgxrdp`.

I'm not as familiar with the workings of `xorgxrdp` as some of the developers, but it strikes me this may be a better default than our existing one.

Thoughts anyone? Are there any drawbacks to doing this?

**Comment by alexanderzhirov** [comment]
I ran into such a problem on my laptop. I made edits according to the specified [post](https://github.com/neutrinolabs/xrdp/issues/2010#issuecomment-1430062495) - the RDP connection began to work correctly.

Configuration edits and setting permissions `660` for `/dev/tty1` did not yield results.

```
Section "Module"
        Load "modesetting"
EndSection

Section "Device"
        Identifier "NVIDIA Card"
        Driver "nvidia"
        BusID "PCI:1:0:0"
        Option "AllowEmptyInitialConfiguration"
        VendorName "NVIDIA Corporation"
        BoardName "GM108M [GeForce 940MX]"
EndSection
```

The RDP connection could not be started from under Nvidia. I used the above solution.
